pkg/option/config_test.go

```diff
@@ -434,6 +441,26 @@ func TestCheckMapSizeLimits(t *testing.T) {
 				WantErr:          true,
 			},
 		},
+		{
+			name: "LB map size below range",
+			d: &DaemonConfig{
+				LBMapEntries: LBMapMinEntries - 1,
+			},
+			want: sizes{
+				LBMapEntries: LBMapMinEntries - 1,
+				WantErr:      true,
+			},
+		},
+		{
+			name: "LB map size above range",
+			d: &DaemonConfig{
+				LBMapEntries: LBMapMaxEntries + 1,
+			},
+			want: sizes{
+				LBMapEntries: LBMapMaxEntries + 1,
+				WantErr:      true,
+			},
```

Comment:
Yeah, because of LBMapEntries. Similar to what we do for `PolicyMapEntries` - https://github.com/cilium/cilium/blob/3466e7157af1949f81fd4ad78f24a4a7817c77c4/pkg/option/config_test.go#L424-L443